features/FEATURE_BLE/targets/TARGET_CORDIO/CordioBLE.h

```diff
@@ -127,6 +128,12 @@ class BLE : public ::BLEInstanceBase {
     virtual void processEvents();
 
 private:
+    /**
+     * Return singleton.
+     * @return GenericGap instance.
+     */
+    const generic::GenericGap& getGenericGap() const;
```

Comment:
This function is not required; you can change `getGap` result to return a `GenericGap`; it is authorized to refine return type in derived classes.

Reply:
avoids cast, functionally same, getGap will inline it so no cost